packages/native-ui/src/hooks/useMedia.ts

```diff
@@ -10,14 +10,15 @@ const useMedia = (): Partial<Record<keyof typeof breakpoints, boolean>> => {
   const mediaBreakpoints: Partial<Record<BreakPointKeys, boolean>> = {};
 
   // Type guard function to check if 'breakpoint' is a key of 'breakpoints'
-  function isBreakpointKey(
-    value: keyof UnistylesBreakpoints,
-    breakpointsObj: typeof breakpoints
-  ): value is BreakPointKeys {
-    return value in breakpointsObj;
-  }
+  // function isBreakpointKey(
+  //   value: keyof UnistylesBreakpoints,
+  //   breakpointsObj: typeof breakpoints
+  // ): value is BreakPointKeys {
+  //   return value in breakpointsObj;
+  // }
 
-  if (isBreakpointKey(breakpoint, breakpoints)) {
+  // if (isBreakpointKey(breakpoint, breakpoints)) {
```

Comment:
@jordmccord I had to do this to make the linter happy after upgrading the deps. How would you like to handle this?

Reply:
What is the lint issue?

Reply:
![Uploading image.png…]()

Reply:
Ok got it 👍